doc/api/magic-plot.md

```diff
@@ -141,6 +141,12 @@ generate histograms without explicitly removing NULL entries.
 %sqlplot histogram --table penguins.csv --column body_mass_g  --bins 100
 ```
 
+### Density curves
+
+```{code-cell} ipython3
```

Comment:
I think we can also add an example in histogram section [here](https://jupysql.ploomber.io/en/latest/plot.html#histogram) to help users understand  when to add the `density` parameter and how it helps with the results. What do you think? @edublancas

Reply:
yeah, I think adding the example will help